server/search/searchDB.js

```diff
@@ -1,8 +1,33 @@
-const {Movie} = require('../database/database');
-
+const {Movie} = require('../database/database.js');
+const redisClient = require('../cacheManager');
+const {transformToSearchDisplay} = require('./searchHelpers')
 
 module.exports = {
-  getMovie: (name) => {
-    // console.log(Movie)
+  getMovie: (searchInputTitle) => {
+    //we need to check cache to see if title is there
+    //this is scuffed
+    //because cache is not set up yet to based on auth.  A sample insert is made
+    //after sample is set ---- comment out
+    // let sampleKey = 'Iron Man 2'
+    // let sampleValue = '10138'
+    // let sample = redis.set(sampleKey, sampleValue)
+    // console.log(sample)
+    const checkCache = redisClient.get(searchInputTitle);
+    //if checkCache returns true, it exists in db - so lets grab it 🚀
+    if (checkCache) {
```

Comment:
this is similar to the function in homeDB, right? ( getMovie: (movieId) ) 
How are they different? It is probably pretty obvious, but I am new to redis.

Reply:
they arent different =) home and search fetch from the same schema to get data to page